libcontainer/rootfs_linux.go

```diff
@@ -594,6 +595,10 @@ func createDevices(config *configs.Config) error {
 }
 
 func bindMountDeviceNode(dest string, node *configs.Device) error {
+	if _, stErr := os.Stat(dest); stErr == nil {
+		// already exist (#2450)
+		return nil
```

Comment:
I don't like this, but if we're going to do it we should actually check that the node matches the configuration (that the type/major/minor is the same). In addition, this should almost certainly be `Lstat`.

Reply:
updated